tarantool/ext_types/packer.py

```diff
@@ -0,0 +1,9 @@
+from decimal import Decimal
+from msgpack import ExtType
+
+import tarantool.ext_types.decimal as ext_decimal
```

Comment:
`ext_types` is a bit confusing. I would rename the directory. Up to you.
```suggestion
import tarantool.mgpack.decimal as ext_decimal
```
or
```suggestion
import tarantool.mgpack.ext.decimal as ext_decimal
```

Reply:
Any other msgpack-related code is scattered across the code so having a separate `msgpack` folder is redundant.

I rename the folder to `msgpack_ext_types`.

Reply:
`msgpack_ext` seems enough. Up to you.

Reply:
Used `msgpack_ext`, thank you